content/en/developers/integrations/create-an-integration-detection-rule.md

```diff
@@ -0,0 +1,146 @@
+3. Verify that the rule is enabled.
+
+## Configuration best practices
+
+In addition to the detection rule definition, the `partnerRuleId` field is required for partner contributed detection rules. The `isEnabled` field should be removed as it does not apply to partner contributed detection rules.
+
+|      | Description    | Examples |
+| ---  | ----------- | ----------- |
+|partnerRuleId | Unique identifier for the rule, following the format `ext-00*-***` where * could be any alphanumeric characters. | `ext-003-bzd` |
+
+### Example of a well-defined detection rule
+
+Selecting a rule type and defining search queries:
+
+{{< img src="developers/integrations/SIEM_detection_rule_top.png" alt="Steps 1-3 of a filled-out detection rule creation form" style="width:90%;" >}}
+
+Setting rule cases and writing the notification message:
+
+{{< img src="developers/integrations/SIEM_detection_rule_bottom.png" alt="Steps 4 and 5 of a filled-out detection rule creation form" style="width:90%;" >}}
+
+For more information, see the documentation on [configuring a detection rule][7].
+
+## Understanding validation messages
+
+### Rule JSON parsing
+```
+File=<FILE_PATH> in collection=<COLLECTION> is an invalid JSON: error=<ERROR>
+```
+This error means that the JSON located at `<FILE_PATH>` is considered invalid JSON
+
+### Rule ID/Rule name
+```
+partnerRuleId is empty for rule name="<RULE_NAME>" - partnerRuleId=<NEW_RULE_ID> is available
+```
+A `partnerRuleId` is required for each rule and is missing. Use the generated `<NEW_RULE_ID>`. 
+
+```
+partnerRuleId=<RULE_ID> is in the incorrect format for rule name="<RULE_NAME>", it must follow the format=^[a-z0-9]{3}-[a-z0-9]{3}-[a-z0-9]{3}$ - partnerRuleId=<NEW_RULE_ID> is available
+```
+The rule name is not in the correct format. Use the generated `partnerRuleId: <NEW_RULE_ID>` to fix the issue.
+
+```
+Duplicate partnerRuleId=<RULE_ID> for rule name="<RULE_NAME>" - <RULE_ID_KEY> must be unique and it is already used in rule_ids="<RULE_IDS>" - <RULE_ID_KEY>=<NEW_RULE_ID> is available
+```
+Each `partnerRuleId` must be unique. The current ID is already being used. The newly generated `partnerRuleId` is available.
+
+```
+Duplicate name="<RULE_NAME>" for <RULE_ID_KEY>=<RULE_ID> - name must be unique.
+```
+Each rule name must be unique. The current name is already being used. Update the rule name to be unique.
+
+### MITRE tags
+```
+The rule with partnerRuleId=<RULE_ID> contains a MITRE tag tactic but it does not contain the tag `security:attack` is not present in, please add it
+```
+When a rule contains a MITRE tag `tactic:<TAG_VALUE>`, the tag `security:attack` must be added to the list of tags.
+
+```
+The MITRE tactic/technique tag=<TAG> for partnerRuleId=<RULE_ID> appears to be incorrect (i.e. it does not exist in the MITRE framework).
+```
+The listed tactic/technique tag `<TAG>` does not follow the [MITRE framework](https://attack.mitre.org/). Please select a different tag.
```

Comment:
I think we should see `Please update the incorrect tag` instead.

Reply:
Should we instead say 'Please select a valid MITRE tag'?